src/RaisedButton/RaisedButton.js

```diff
@@ -251,21 +251,22 @@ class RaisedButton extends Component {
 
   state = {
     hovered: false,
+    keyboardFocused: false,
     touched: false,
     initialZDepth: 0,
     zDepth: 0,
   };
 
   componentWillMount() {
-    const zDepth = this.props.disabled ? 0 : 1;
+    const zDepth = (this.props.disabled) ? 0 : 1;
```

Comment:
I couldn't find an eslint rule, but when I went back to remove them earler, they did seem like overkill on second read. Where they are certainly more useful is with compound conditions (a === b && x === y). I've left the one instance of that type with parens.